src/components/map-container.js

```diff
@@ -459,6 +470,22 @@ export default function MapContainerFactory(MapPopover, MapControl) {
           >
             {this._renderDeckOverlay(layersToRender)}
             {this._renderMapboxOverlays(layersToRender)}
+            {/*
+                By placing the editor in this map we have to perform fewer checks for css zIndex
+                and fewer updates when we switch from edit to read mode
+              */}
+            <Draw
+              datasets={datasets}
+              editor={uiState.editor}
+              features={editor.features}
+              isEnabled={isEdit}
+              layers={layers}
+              onDeleteFeature={uiStateActions.deleteFeature}
+              onSelect={uiStateActions.setSelectedFeature}
+              onUpdate={visStateActions.setFeatures}
+              style={{zIndex: isEdit ? 0 : -1}}
```

Comment:
making zIndex to -1 when !isEdit cauing deckgl overlay showing on top of the draw element
<img width="479" alt="Screen Shot 2019-12-05 at 3 58 53 PM" src="https://user-images.githubusercontent.com/3605556/70284235-40dd9800-1778-11ea-8cb6-81444addeb75.png">

Reply:
changing the z-index, in this case, will not fix the issue because all layers are rendered onto a canvas and the editor  is a regular html element.
In order to have the effect you describe we need to absolute position the editor container but in that case deck.gl will not receive mouse over and the tooltip will not show up